pkg/handlers/health_check_test.go

```diff
@@ -18,4 +18,7 @@ func (s HandlerTestSuite) TestHealthcheckHandler() {
 
 	s.NoError(err)
 	s.Equal(statusPass, healthCheckActual.Status)
+	s.Equal("", healthCheckActual.Datetime)
```

Comment:
You should be able to set these and test against actual values, not empty strings.

Reply:
That would be ideal. What would be the best way I could mock the environment?

Reply:
Would `os.Setenv` be okay for this part?

Reply:
If you do something like this, you should be able to directly modify them in the test setup and not have to do anything with the env variables themselves.